_layouts/report.md

```diff
@@ -147,5 +155,21 @@
             </div>
             <br><br>
         </div>
+        <script>
+        $('document').ready(function(){
+            var hash_value = document.getElementById('sha256sum_hash_value').textContent.trim();
+            document.getElementById('sha256sum_hash_value').textContent = hash_value.substring(0,7);
+            document.querySelector(".fa-clipboard").addEventListener("click", function(){
```

Comment:
you use single quotes and double quotes in JS ... which one should be used?  consistency is important.

Reply:
Since there is a choice to programmer which quoting style he prefers in JavaScript code. To maintain the consistency over all `.js` files I will make changes to single quotes 😃